package.json

```diff
@@ -35,19 +35,21 @@
     "@backbase/arrangement-manager-http-ang": "^1.3.7",
     "@backbase/backbase-theme": "^2.316.0",
```

Comment:
With ui-ang 8 the theme should be imported from ui-ang directly and this dependency removed

Reply:
I can't see this package in dependencies for `ui-ang`. Is it a task to do?

Reply:
they basically embedded the theme in ui-ang, so the backbase-theme package will not be published anymore from ui-ang 8 onwards, but the sass files will be part of ui-ang itself

Reply:
After removing theme package and applying suggested changes I received a broken layout:

<img width="843" alt="image" src="https://user-images.githubusercontent.com/1961590/181737965-1bc1fd0a-ee05-422b-b094-5efbef4da792.png">

<img width="629" alt="image" src="https://user-images.githubusercontent.com/1961590/181738027-8b63facc-bd9b-48af-9c11-8d837a6ae7a7.png">

I've fixed it by overriding `z-index` for the `bb-layout__topbar`: https://github.com/Backbase/golden-sample-app/pull/54/commits/fecd0f9714588f1eeae23e597d0953b5f5c8ab43#diff-c0301706b3c2178287213cb76542bff338c966bf41db75421789a9be0efc58f9R16

```
.bb-layout .bb-layout__topbar {
  z-index: $zindex-modal-backdrop + 1;
}
```

Reply:
FYI @okadurin

Reply:
Odd I didn't have to do that in my branch 🤷